CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ v5.18.0
 
 * Add Location Control sources of information `<https://github.com/lsst-ts/LOVE-frontend/pull/446>`_
 * Fix LATISS and ATCamera state mappings `<https://github.com/lsst-ts/LOVE-frontend/pull/445>`_
+* Add ``Aircraft Tracker`` component `https://github.com/lsst-ts/LOVE-frontend/pull/447`
```

Comment:
This should be just the same title of the PR. I recommend ti remove the "Tickets/love 169" prefix as that information is stored on the branch name and we only want to give a more natural description.